README.md

```diff
@@ -104,6 +113,15 @@ err := client.Publish(
 * SpanContext : Opentracing Jaeger Span Context(string - optional)
 * Context : Golang context. (context - default: context.background)
 * Version : Version of schema. (integer - default: `1`)
+* EventID : Event ID. Used by GDPR service. (integer)
```

Comment:
I think we should not mention GDPR service because there is a possibility other services than GDPR service still used that. Maybe just mention backward compatibility.

Reply:
let's update

Reply:
@leowind updated